packages/tools/devtools/devtools-core/src/messaging/container-devtools-messages/DataEdit.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { HasContainerKey } from "../../CommonInterfaces";
+import { SharedObjectEdit } from "../../data-visualization";
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link DataEdit.Message}.
+ *
+ * @internal
+ */
+export namespace DataEdit {
+	/**
+	 * {@link DataEdit.Message} {@link IDevtoolsMessage."type"}.
+	 *
+	 * @internal
+	 */
+	export const MessageType = "SEND_EDIT_DATA";
```

Comment:
We should rename this string to match.
```suggestion
	export const MessageType = "DATA_EDIT";
```